eman2/convert/dataimport.py

```diff
@@ -31,7 +31,7 @@
 import pyworkflow.utils as pwutils
 from pwem.objects.data import Coordinate, CTFModel
 from pwem.objects.data_tiltpairs import Angles
-from xmippLib import MetaData, MDL_XCOOR, MDL_YCOOR, MDL_PICKING_PARTICLE_SIZE
+from pwem.metadata import MetaData, MDL_XCOOR, MDL_YCOOR, MDL_PICKING_PARTICLE_SIZE
```

Comment:
I like it but I don't remember if tomo people wanted xmippLib on purpose here

Reply:
Thanks Gregory. We'll work on it by the time the tomo part is brought to scipion3.